validator/src/main/resources/expected-data-template/eksContainerInsightExpectedMetrics.mustache

```diff
@@ -184,3 +184,464 @@
       value: 2
     - name: envoy_http_conn_manager_prefix
       value: ingress
+
```

Comment:
So one test run contains all workload and expect all the metrics (memcache, jmx, nginx etc.) to show up?

Reply:
Yes